backend/api/routes/routes.go

```diff
@@ -131,6 +133,12 @@ func SetupRoutes(router *gin.Engine) {
 	eventRoutes := router.Group("/api/event")
 	eventRoutes.Use(middleware.AuthMiddleware())
 	{
+		// POST api/event/Announcements/:announcement_id
+		eventRoutes.POST("/announcements/:announcement_id", event.PostAnnouncementHandler)
```

Comment:
We do not need the announcement_id in this route as it is created when the endpoint is hit.

Reply:
I'm pretty sure I need to keep the ending slash there...